Position.hpp

```diff
@@ -133,22 +133,22 @@ class Position {
   /**
    * return true if current player can win next move
    */
-  bool canWinNext() const {
+  inline bool canWinNext() const {
     return winning_position() & possible();
```

Comment:
the c++ inline keyword does not actually tell the compiler to inline the function, it is rather used to avoid linkage conflict which does not appear in the current code. https://en.cppreference.com/w/cpp/language/inline

I will add the inline qualifier when it will be needed.

Reply:
Okay, understood